pywhy_graphs/testing.py

```diff
@@ -0,0 +1,27 @@
+import networkx as nx
+
+
+def assert_mixed_edge_graphs_isomorphic(G1, G2):
```

Comment:
This looks good, I presume all roundtrip tests should use this testing function ?

Reply:
Yep!

FYI I realized there's a lot of LOC scattered in the codebase doing this.